.circleci/config.yml

```diff
@@ -43,7 +43,6 @@ commands:
           name: Generate CCI cache key
           command: |
             echo "$(date "+%D")" > .cachekey
-            cat .circleci/cached_datasets_list.txt >> .cachekey
       - persist_to_workspace:
           root: .
           paths:
```

Comment:
hmm, it seems used in caching vectors. but let me take a deeper look, we might be able to completely get rid of caching.

Reply:
It looks like the caching steps have been failing for [Linux](https://app.circleci.com/pipelines/github/pytorch/text/5197/workflows/b70e051b-845e-4543-920c-a201e25ac198/jobs/174614), [MacOS](https://app.circleci.com/pipelines/github/pytorch/text/5197/workflows/b70e051b-845e-4543-920c-a201e25ac198/jobs/174592), and [Windows](https://app.circleci.com/pipelines/github/pytorch/text/5197/workflows/b70e051b-845e-4543-920c-a201e25ac198/jobs/174601) unittests.

Reply:
just had a look, unfortunately, it is not trivial to remove vectors caching. it is used in several places in https://github.com/pytorch/text/blob/main/test/test_build.py. I am not sure if removing the caching would significantly increase the test time?

The problem with these tests is exactly the same as with datasets before mocking i.e they are using real data to test.

Reply:
Gotcha, in that case can we figure out how to fix this error for the time being?

```
error computing cache key: template: cacheKey:1:27: executing "cacheKey" at <checksum ".cachekey">: error calling checksum: open /root/project/.cachekey: no such file or directory
```

Could this be because we don't explicitly create the `/root/project/.cachekey` directory? I wonder if our unittests are even able to take advantage of the cache folder right now.

Reply:
yupp, I made changes to resemble to original cache before introducing the caching for datasets. 

> I wonder if our unittests are even able to take advantage of the cache folder right now.
Right, I mistakenly got rid of all the caching when removing caching for datasets in this PR https://github.com/pytorch/text/pull/1587. I guess, we might be able to remove the cache completely and just do testing without it, although I think it wouldn't hurt to have caching either. Let's land this PR to resolve the above issue, and we can follow-up later to see if we want to completely remove caching.